src/test/regress/expected/upgrade_citus_finish_citus_upgrade.out

```diff
@@ -0,0 +1,14 @@
+-- Citus upgrades are finished by calling a procedure
+-- this is a transactional procedure, so rollback should be fine
+BEGIN;
+	CALL citus_finish_citus_upgrade();
+NOTICE:  Preparing all the existing partitioned table indexes
+NOTICE:  Preparing to sync the metadata to all nodes
+ROLLBACK;
+-- do the actual job
+CALL citus_finish_citus_upgrade();
+NOTICE:  Preparing all the existing partitioned table indexes
+NOTICE:  Preparing to sync the metadata to all nodes
+-- idempotent, should be called multiple times
+CALL citus_finish_citus_upgrade();
+NOTICE:  already at the latest distributed schema version (11.1-1)
```

Comment:
wil this become something we have to change in the future?

Reply:
That is true, it'll change. I can increase the log level to warning here